ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -36,11 +36,10 @@ Item {
 
     property bool isSectionActive: mainModule.activeSection.id === parentModule.getMySectionId()
     property string activeChatId: parentModule && parentModule.activeItem.id
-    property string activeSubItemId: parentModule && parentModule.activeItem.activeSubItem.id
     property int chatsCount: parentModule && parentModule.model ? parentModule.model.count : 0
     property string activeChatType: parentModule && parentModule.activeItem.type
     property bool stickersLoaded: false
-    property var contactDetails: Utils.getContactDetailsAsJson(root.activeChatId, false)
+    property var contactDetails: activeChatType === Constants.chatType.oneToOne && Utils.getContactDetailsAsJson(root.activeChatId, false)
```

Comment:
This seems wrong, it's not a `bool`

Reply:
This is black magic Javascript. It will only evaluate and use the second part if the first one is `true`.
Equivalent to 
```js
activeChatType === Constants.chatType.oneToOne ? Utils.getContactDetailsAsJson(root.activeChatId, false) : null
```